esp/src/eclwatch/WUDetailsWidget.js

```diff
@@ -234,6 +236,15 @@ define([
                 });
                 this.wu.refresh();
             }
+            WsWorkunits.WUInfo({
+                request: {
+                    Wuid: this.wu.Wuid
+                }
+            }).then(function (response) {
+                if (response.WUInfoResponse.SecMethod === null) {
+                    query(".scopeOptional").forEach(domConstruct.destroy);
```

Comment:
To not to confuse the user we do not display it Kevin's recent work or in this one. If they are not in a authenticated set up this field has no relevance.